docs/topics/modules/viewing_projects_oso.adoc

```diff
@@ -1,7 +1,7 @@
 [id="viewing_projects_oso"]
 = Viewing projects in OpenShift Online
 
-To view the OpenShift Online projects that support the pipeline for your project, navigate to https://console.starter-us-east-2.openshift.com/.
+To view the OpenShift Online projects that support your project's pipeline, navigate to link:https://console.starter-us-east-2.openshift.com/[console.starter-us-east-2.openshift.com].
```

Comment:
Please, don't use possessive with inanimate things.

"that support the pipeline for your project" is fine.